### PR TITLE
Add instructions for setting up UFW/firewall rules and fix some typos.

### PR DESCRIPTION
**Pull request by sjoerdschouten**
Add firewall / ufw instructions to run noson with an enabled firewall. See thread: https://github.com/janbar/noson-app/issues/121

### CODE REVIEW

**Comment by baurmatt** [comment]
Just tested this... Sadly this doesn't work :(

**Comment by baurmatt** [comment]
@sjoerdschouten I think what is missing, is the 40000:60000 UDP port range suggested by @jankohoutek in #121. This made it work for me.

File: README.md [hunk 1]
```diff
@@ -88,17 +88,37 @@ Launch the *Noson* application from command line as follows.
...
-In some circonstances your network couldn't allow broadcast/multicast traffic. To workaround the limitation you can configure the application by specifying a Sonos device endpoint.
+**To add the according to [Sonos](https://support.sonos.com/s/article/688?language=en_US), required rules to `ufw` you can use the following instructions :**.
+
+- Create a file named `noson` in /etc/ufw/applications.d` with the following:
+
+``` ini
+[noson]
+title=noson Sonos controller app
+description=controls Sonos devices on the same network
+ports=80,443,445,1400:1410,3400,3401,3405,3445,3500,4070,4444/tcp|136,137,138,139,1900,1901,2869,10243,10280,10281,10282,10283,10284,5353,6969/udp|35382
```
Comment (baurmatt) [inline README.md:109]:
```suggestion
ports=80,443,445,1400:1410,3400,3401,3405,3445,3500,4070,4444/tcp|136,137,138,139,1900,1901,2869,10243,10280,10281,10282,10283,10284,5353,6969/udp|40000:60000/udp|35382
```

For me, UDP port 40000-60000 port was needed to make it work. The format looks kind of strange, but this seems to be required. Adding the port range to the existing UDP ports led to the error message:

```
WARN: "Invalid ports in profile 'noson'"
```

**Comment by janbar** [comment]
Thanks

File: README.md (hunk 1 above)
Comment (baahver) [reply README.md:109]: This is working for me too. adding |40000:60000/udp| did the trick. Isn't it strange that it extending udp ports does not work.
`|136,137,138,139,1900,1901,2869,10243,10280,10281,10282,10283,10284,5353,6969,40000:60000/udp|35382`

**Comment by sarvasana** [comment]
This does not work.

**Pull request merged**